chargeOclock/Source/Models/Client.swift

```diff
@@ -14,8 +14,8 @@ protocol ClientSettable {
 
 class Client {
 
-	var id: Int = 0
-	var isDirty: Bool = false
+	var id: Int? = nil
+	var isDirty: Bool!
```

Comment:
not sure what you're talking about... a Bool can't be uninitialised, it's either true or false...
and it's set in the constructors 🤔